hr_attendance_modification_tracking/views/hr_attendance_view.xml

```diff
@@ -19,4 +23,33 @@
         </field>
     </record>
 
+<!--  insert in tree view 'Time Changed' new field  -->
+       <record id="hr_attendance_view_tree" model="ir.ui.view">
+        <field name="name">hr.attendance.view.tree</field>
+        <field name="model">hr.attendance</field>
+        <field name="type">tree</field>
+        <field name="inherit_id" ref="hr_attendance.view_attendance_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='check_out']" position="after">
+                     <field name="time_changed_manually"/>
+            </xpath>
+        </field>
+    </record>
+
+<!--  insert in search filter and Group by usint a new field 'Time Changed'  -->
+    	<record id="project_view_search_inherit" model="ir.ui.view">
+		<field name="name">hr.attendance.inherited.view</field>
+		<field name="model">hr.attendance</field>
+		<field name="inherit_id" ref="hr_attendance.hr_attendance_view_filter"/>
+		<field name="arch" type="xml">
+            <filter name="nocheckout" position="after">
```

Comment:
Here I dont know if this is working but name attribute is kind of an identifier so it would be better of there arent two filters with the same name. Maybe you can call the second `name="group_by_time_changed_manually"`

Reply:
Of course, done. Thanks.